submodules/power_management_services_setup/README.md

```diff
@@ -0,0 +1,58 @@
+# Submodule power_management_services_setup
```

Comment:
This module seems to have very specific expectations in term of flavor of vsi for jump box and services (eg: suse), but 
1. This is not documented explicitly anywhere
2. There is no validation block (remote-exec) that the hosts are aligned with requirement (eg: could do a uname -a, etc to validate and return a nice error message)

Reply:
It'd also be good to have one of the example using the service_setup, currently there is no example, nor tests against that logic.

Reply:
- standard and imm-catalog/standard-solution examples utilise management submodule.
- the only use case where management submodule is not used is the basic example (due to limitations in landscape prerequisites, like missing transit gateway, and VSIs on VPC)
- setup management services --> proxy is more or less mandatory, since otherwise you do not have access to public internet from PowerVS.

Reply:
@Beilmann - It'd be good to have this in one example that is run through ci pipeline so that regressions can be caught early in this area. I do not think ibm-catalog/standard-solution is currently run in ci pipeline (not referenced from https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/blob/develop/tests/pr_test.go) so either we add this to the pr_test or create an additional example leveraging the slz module. The slz module is already available at https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone

Reply:
We could have this example / test in a subsequent PR, I do not think this block this one. We should just have some more documentation and checks on expectations in this PR tough.

Reply:
Hmm right.

Reply:
@surajsbharadwaj - could the action be tracked in an issue ticket at https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker so that it does not fall through the cracks?

Reply:
@Beilmann can you comment here ?

Reply:
@surajsbharadwaj - track the issue for extending basic example to deploy SLZ available in public git and put powerVS infrastructure on top of it. Then we can use cloud connections and also deploy management services.